networklibrary/src/main/java/com/eis/smsnetwork/SMSNetworkManager.java

```diff
@@ -115,6 +174,33 @@ public void setNetSubscriberList(@NonNull NetSubscriberList<SMSPeer> list) {
      * @param dictionary A NetDictionary of type <String,String> to provide
      */
     public void setNetDictionary(@NonNull NetDictionary<String, String> dictionary) {
+        netDictionary = dictionary;
+    }
 
+    /**
+     * Setups all the basic android-related operations to let the network function
+     *
+     * @param context The android context to make this work on
+     */
+    public void setup(Context context) {
+        SMSManager.getInstance().setReceivedListener(BroadcastReceiver.class,
+                context.getApplicationContext());
+        SMSMessageHandler.getInstance()
+                .setMessageParseStrategy(new MessageParseStrategy<String, SMSPeer, SMSMessage>() {
+                    private String hiddenCharacter = "¤";
```

Comment:
Kind of hardwiring?

Reply:
I actually have no idea why that symbol is checked...
![image](https://user-images.githubusercontent.com/22344699/73648654-4f640500-467e-11ea-8b35-3a5b93ecc2b1.png)

Reply:
Oh no wait, that's not what this is doing.
Basically it's used instead of the \r for the smslibrary to still keep a gsm character set.
So idk, I wouldn't really consider it an hardwiring, but I totally see why you would think it is.

Reply:
@CremaLuca I don't get why this would be an hardwiring, it's the same as in [DefaultSMSMessageParseStrategy](https://github.com/Cogno-IDU/smslibrary/blob/master/smslibrary/src/main/java/com/eis/smslibrary/SMSMessageHandler.java)

Reply:
Yeah I didn't mean hardwiring as something that could be asked as parameter, I just prefer the character would be placed inside a constant.

Reply:
Right, we should make it final